node/network/statement-distribution/src/tests.rs

```diff
@@ -1842,6 +1842,324 @@ fn peer_cant_flood_with_large_statements() {
 	executor::block_on(future::join(test_fut, bg));
 }
 
+// This test addresses an issue when received knowledge is not updated on a
```

Comment:
We should elaborate the issue observed here, `an issue` will not be enough in 3 months time.

Reply:
We should create one, I am not aware of one at this time.

Reply:
I have linked this PR for reference.